### PR TITLE
Bsw 3241 faqs terms

### PR DESCRIPTION
**Pull request by den-d4**

### CODE REVIEW

File: src/components/Faqs/FaqAccordion/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }
+`;
+
+const StyledChevronIcon = styled(({ ...props }) => <ChevronDownIcon {...props} />)<{ isOpen?: boolean }>`
+  transform: scale(${({ isOpen }) => (isOpen ? "1, -1" : "1, 1")});
+  transition: transform 0.3s ease;
+`;
+
+const Answer = styled(Box)<{ isOpen: boolean }>`
+  height: ${({ isOpen }) => !isOpen && "0"};
```
Comment (chefReyMysterio) [inline src/components/Faqs/FaqAccordion/index.tsx]: don't use && operator in css. use ternary condition

File: src/components/Terms/TermsAccordion/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  justify-content: space-between;
+  cursor: pointer;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    width: max-content;
+  }
+`;
+
+const TermsWrapper = styled(Box)<{ isOpen: boolean }>`
+  height: ${({ isOpen }) => !isOpen && "0"};
```
Comment (chefReyMysterio) [inline src/components/Terms/TermsAccordion/index.tsx]: same &&

File: src/components/Faqs/FaqAccordion/index.tsx (hunk 1 above)
Comment (den-d4) [reply src/components/Faqs/FaqAccordion/index.tsx]: fixed

File: src/components/Terms/TermsAccordion/index.tsx (hunk 2 above)
Comment (den-d4) [reply src/components/Terms/TermsAccordion/index.tsx]: fixed

**Review by devStieg — APPROVED** [review]

**Pull request merged**